bower.json

```diff
@@ -8,6 +8,7 @@
   "main": "quick-element.html",
   "license": "MIT",
   "dependencies": {
-    "polymer": "polymer/polymer#0.8-preview"
+    "polymer": "polymer/polymer#0.8-preview",
+    "quick-element": "polymer/quick-element"
```

Comment:
I don't understand the part about quick-element dependency being skipped. Also, with the original version, the context is neutral. The only thing that matters is that all dependencies are in sibling folders.